src/main/java/com/google/sps/servlets/LiveCourseData.java

```diff
@@ -36,14 +36,16 @@ public TermDataHolder getAllDataFromTerm(DatastoreService db, HttpServletRequest
 
     termDataHolder.setHoursList(getDataFromTermRating(db, termKey, "hours"));
     termDataHolder.setDifficultyList(getDataFromTermRating(db, termKey, "difficulty"));
-    termDataHolder.setTermScoreList(getDataFromTermRating(db, termKey, "score-term"));
-    termDataHolder.setTermPerceptionList(getDataFromTermRating(db, termKey, "perception-term"));
+    termDataHolder.setTermScoreList(getDataFromTermRating(db, termKey, "perception-term"));
+    termDataHolder.setTermPerceptionList(getDataFromTermRating(db, termKey, "score-term"));
```

Comment:
Why are we swapping terms here but not what we're setting?

Reply:
This is due to a misunderstanding I had with Laura on what each thing meant. When she creates the rating entity, the score-term property on the rating entity actually refers to the perception of a term because the API calls it a "perception score"  and the "perception-term" property relates the the term-score. It's kinda backwards so I mixed them up, but because we already have some seeded data I went for this quick fix rather than having her rename the entity property

Reply:
Hmm. Well it's definitely super confusing to look at. Is there a TODO somewhere to fix this? It'd be good to get this finished before reviewers look at the completed code/etc.

Reply:
I will speak with Laura first thing tomorrow and get it fixed